packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1116,11 +1137,12 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return !this.disableIsolatedChannels || this.gcEnabled;
     }
 
-    private formMetadata(): IContainerRuntimeMetadata {
+    private formMetadata(): WriteContainerRuntimeMetadata {
```

Comment:
The PR description says `summaryFormatVersion` is updated to 2. But I think you went with option (2) mentioned in the description?

Reply:
Oh yes, I forgot to update the description afterwards, I did go with option (2).

Reply:
Updated the description to match.